components/brave_rewards/resources/extension/brave_rewards/background/api/locale_api.ts

```diff
@@ -90,7 +92,11 @@ export const getUIMessages = (): Record<string, string> => {
     'tipsProcessedNotification',
     'tokenGrant',
     'tokenGrantClaimed',
+<<<<<<< HEAD
     'unVerifiedCheck',
+=======
+    'turnOnAds',
+>>>>>>> Fixes brave-browser/issues#2914, adds launch notification for Rewards users about Ads
```

Comment:
you probably already know, but FYI there's a merge conflict here ^

Reply:
yep :) fixed